src/features/cookies/atoms/CloseButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { t } from '@lingui/macro'
+import React from 'react'
+import styled from 'styled-components/native'
+
+import { accessibilityAndTestId } from 'libs/accessibilityAndTestId'
+import { styledButton } from 'ui/components/buttons/styledButton'
+import { Touchable } from 'ui/components/touchable/Touchable'
+import { Invalidate } from 'ui/svg/icons/Invalidate'
+import { Typo, Spacer } from 'ui/theme'
+
+interface Props {
+  onPress?: () => void
```

Comment:
je pense que si on met un bouton, il **doit** faire une action lorsqu'on clic dessus

```suggestion
  onPress: () => void
```

Reply:
Ce composant va être supprimé